Automattic/woocommerce-payments
# Updated currency symbols to display correctly for enabled currencies.

Fixes #1922

#### Changes proposed in this Pull Request

* Updated areas that display currency symbols to display them using `decodeEntities` from the `html-entities` package.
* Deconstructed `currency` prop in `EnabledCurrenciesListItem` for better code.

#### Testing instructions

* Switch to the branch.
* Navigate to WooCommerce > Settings > Multi-Currency.
* Enabled currencies list should have currency symbols and not html entities.
* Click on Add Currencies button.
* Checkbox items should have currency symbols and not html entities.

-------------------
